src/utils/fileUpload.ts

```diff
@@ -13,7 +15,7 @@ export const uploadFileToStorage = async (
   formData.append("file", fileToUpload);
 
   const response = await fetch(
-    `/api/v3/file/upload/${schoolId}/boardnodes/${roomId}`,
+    `${FILE_API_UPLOAD}${schoolId}/${FILE_API_BOARDNODES}/${roomId}`,
```

Comment:
What do you think about:
FILE_API_UPLOAD= "/api/v3/file/upload/SCHOOLID/boardnodes/PARENTID';
const url = FILE_API_UPLOAD.replace('SCHOOLID', schoolId).replace('PARENTID', roomId);
const response = await fetch(url);

Is this easier, or more unclear?